funcbench/main.go

```diff
@@ -50,43 +51,67 @@ func (l *logger) FatalError(err error) {
 }
 
 func main() {
+	cfg := struct {
```

Comment:
What about using something smarter than this: https://github.com/bwplotka/flagarize

((((:

Reply:
I'd like to try this with funcbench as it has all the keys that we're using and will make it look real clean :sparkles:  , but if we decide to use this, It'll be good to do it for other tools in test-infra aswell to have some consistency. But since flagarize is super new, wonder what @krasi-georgiev thinks?